tests/backup/backup_test.go

```diff
@@ -7075,6 +7075,134 @@ var _ = Describe("{RestartBackupPodDuringBackupSharing}", func() {
 	})
 })
 
+// This testcase verifies schedule backup creation with a single namespace.
+var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
+	var contexts []*scheduler.Context
+	var appContexts []*scheduler.Context
+	var backupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var bkpNamespaces []string
+	var cloudAccountName string
+	var backupName string
+	var schBackupName string
+	var schPolicyUid string
+	var restoreName string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var testrailID = 58014 // testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/58014
+	namespaceMapping := make(map[string]string)
+	labelSelectors := make(map[string]string)
+	cloudCredUIDMap := make(map[string]string)
+	backupLocationMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	timeStamp := strconv.Itoa(int(time.Now().Unix()))
+	periodicPolicyName := fmt.Sprintf("%s-%s", "periodic", timeStamp)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScheduleBackupCreation", "Create schedule backup creation with a single namespace", nil, testrailID)
+		log.Info("Application installation")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Schedule Backup Creation with single namespace", func() {
+		Step("Validate deployed applications", func() {
+			ValidateApplications(contexts)
+		})
+
+		Step("Adding Cloud Account", func() {
+			log.InfoD("Adding cloud account")
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudAccountName = fmt.Sprintf("%s-%v", provider, timeStamp)
+				cloudCredUID = uuid.New()
+				cloudCredUIDMap[cloudCredUID] = cloudAccountName
+				CreateCloudCredential(provider, cloudAccountName, cloudCredUID, orgID)
+			}
+		})
+
+		Step("Adding Backup Location", func() {
+			providers := getProviders()
+			for _, provider := range providers {
+				cloudAccountName = fmt.Sprintf("%s-%v", provider, timeStamp)
+				backupLocationName = fmt.Sprintf("auto-bl-%v", time.Now().Unix())
+				backupLocationUID = uuid.New()
+				backupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateBackupLocation(provider, backupLocationName, backupLocationUID, cloudAccountName, cloudCredUID,
+					getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, "Adding Backup Location")
```

Comment:
Here we can add the backup location name also which will help in debugging in case of failure

Reply:
Done